src/app/pages/viewing/messages/messages.component.ts

```diff
@@ -58,22 +58,25 @@ export class MessagesComponent implements OnInit {
     const bgB = parseInt(backColor.slice(5), 16);
     let headerTextColor;
     let trashIcon;
+    let bookmarkIcon;
     let popupIcon;
```

Comment:
```suggestion
    let headerTextColor: string;
    let trashIcon: string;
    let bookmarkIcon: string;
    let popupIcon: string;
```
Change as necessary if the types are wrong.